src/descriptor/mod.rs

```diff
@@ -535,8 +537,10 @@ impl<'de, Pk, Pkh> de::Deserialize<'de> for Descriptor<Pk, Pkh> where
         struct StrVisitor<Qk, Qkh>(PhantomData<(Qk, Qkh)>);
 
         impl<'de, Qk, Qkh> de::Visitor<'de> for StrVisitor<Qk, Qkh> where
-            Qk: fmt::Debug + str::FromStr,
-            Qkh: fmt::Debug + str::FromStr,
+            Qk: fmt::Debug + str::FromStr + serde::export::fmt::Display
+            + serde::export::Clone,
```

Comment:
Please just use `fmt::Display` and `Clone`. Using the serde exports for stdlib things feels really strange.

Reply:
Don't really think when I did it this way. The compiler told me to use these imports.